INSTALL

```diff
@@ -26,3 +26,15 @@ Brief Instructions
    with unixODBC 2.2.11.
 
    For any other information please see the Users Guide.
+
+Docker installation (for Debian-based images)
+
+    RUN apt install -y git automake autoconf libtool make gcc perl gettext unixodbc-dev
```

Comment:
What is `RUN` ? Means run a command or is a Docker prompt? Usually I prefer the classic `$` for normal user and `#` for root